subsys/bluetooth/controller/ll_sw/ull_adv_internal.h

```diff
@@ -1,23 +1,64 @@
 /*
- * Copyright (c) 2017-2019 Nordic Semiconductor ASA
+ * Copyright (c) 2017-2020 Nordic Semiconductor ASA
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#define ULL_ADV_RANDOM_DELAY HAL_TICKER_US_TO_TICKS(10000)
+
 int ull_adv_init(void);
 int ull_adv_reset(void);
 
 /* Return ll_adv_set context (unconditional) */
-struct ll_adv_set *ull_adv_set_get(u16_t handle);
+struct ll_adv_set *ull_adv_set_get(u8_t handle);
 
 /* Return the adv set handle given the adv set instance */
 u16_t ull_adv_handle_get(struct ll_adv_set *adv);
 
 /* Return ll_adv_set context if enabled */
-struct ll_adv_set *ull_adv_is_enabled_get(u16_t handle);
+struct ll_adv_set *ull_adv_is_enabled_get(u8_t handle);
 
 /* Return flags, for now just: enabled */
-u32_t ull_adv_is_enabled(u16_t handle);
+u32_t ull_adv_is_enabled(u8_t handle);
 
 /* Return filter policy used */
-u32_t ull_adv_filter_pol_get(u16_t handle);
+u32_t ull_adv_filter_pol_get(u8_t handle);
+
+/* Return ll_adv_set context if created */
+struct ll_adv_set *ull_adv_is_created_get(u8_t handle);
+
+/* Helper function to construct AD data */
+u8_t adv_data_set(struct ll_adv_set *adv, u8_t len, u8_t const *const data);
+
+/* Helper function to construct SR data */
+u8_t scan_rsp_set(struct ll_adv_set *adv, u8_t len, u8_t const *const data);
```

Comment:
I think this function name should have `ull_` as prefix.

Reply:
addressed.